api/ring-camera.ts

```diff
@@ -46,6 +47,19 @@ function getBatteryLevel(data: CameraData) {
   return batteryLevel
 }
 
+function getExistingDoorbellType(data: CameraData): DoorbellType | undefined {
```

Comment:
What do you think about using a different name? `Existing` is a bit vague, but I'm not sure what would be better.  `Physical`, `Mechanical`, `InHome`, `HardWired`...any other ideas?

Reply:
I agree! Switched the name to `InHome`, since it matches the text in the Ring app UI (“Ring my in-home doorbell”): https://github.com/dgreif/ring/pull/152/commits/8db456b598f1616431dd0c1158d5cfd641a9fe05